src/Application/Regulation/View/NamedStreetView.php

```diff
@@ -4,14 +4,13 @@
 
 namespace App\Application\Regulation\View;
 
-final readonly class LocationView
+final readonly class NamedStreetView implements LocationViewInterface
 {
     public function __construct(
+        public string $roadType,
```

Comment:
Est-ce qu'on ne devrait pas hardcoder le roadType plutôt que risquer que le code appelant mette la mauvaise valeur ?

```suggestion
    public string $roadType = RoadTypeEnum::NAMED_STREET->value;

    public function __construct(
```